binding/ruby/test/core/test-headers.rb

```diff
@@ -10,8 +10,8 @@ def test_compare
     header1 = Milter::Header.new("name1", "value1")
     header2 = Milter::Header.new("name2", "value2")
     assert_equal(0, header1 <=> header1)
-    assert_equal(-1, header1 <=> header2)
-    assert_equal(1, header2 <=> header1)
+    assert_true((header1 <=> header2) < 0)
```

Comment:
We should use power assert for easy to debug message:

```suggestion
    assert do
      (header1 <=> header2) < 0
    end
```